lib/skate/oban/clean_up_notifications.ex

```diff
@@ -0,0 +1,46 @@
+defmodule Skate.Oban.CleanUpNotifications do
+  @moduledoc """
+  Cleans up arg:`limit` records older than arg:`cutoff_days` days.
+  """
+
+  require Logger
+
+  use Oban.Worker,
+    queue: :default,
+    unique: [period: 300]
+
+  import Ecto.Query
+
+  @seconds_per_day 24 * 60 * 60
+
+  @impl Oban.Worker
+  def perform(%Oban.Job{args: args}) do
+    cutoff_days = Map.get(args, "cutoff_days", 100)
+    limit = Map.get(args, "limit", 100)
+
+    oldest_date = DateTime.utc_now() |> DateTime.add(-cutoff_days * @seconds_per_day)
+
+    Logger.notice("#{__MODULE__} starting cleanup")
+
+    {time, {count, nil}} =
+      :timer.tc(fn ->
+        from(notification_indexed in Notifications.Db.Notification,
+          where:
+            notification_indexed.id in subquery(
+              from(notification_limit in Notifications.Db.Notification,
```

Comment:
**nitpick:** I think calling this `notification_limited` would be a bit clearer (since it's the notifications that have been limited, not the limit of how many notifications to process), and would also match with `notification_indexed`.

Reply:
bafaeccf47ea9b54665b0f598d792ea6f928ffb6